framework/math/functions/function.h

```diff
@@ -12,6 +12,7 @@ namespace opensn
 class Function : public Object
 {
 public:
+  Function(void) = default;
```

Comment:
We are not using `void` when functions do not take parameters. So, remove it from here and all the other instances as well.

Reply:
it has been fixed